int/api/cmd/execute_function.go

```diff
@@ -38,9 +39,17 @@ func ExecuteFunctionHandler(params operations.CmdExecuteFunctionParams, app *fyn
 			})
 	}
 
-	password := gui.AskPassword(wallet.Nickname, app)
+	password, status := gui.AskPassword(wallet.Nickname, app)
+	if !status {
+		return createInternalServerError(websites.ErrorCodeWalletCanceledAction, websites.ErrorCodeWalletCanceledAction)
+	}
 
 	err = wallet.Unprotect(password, 0)
+
+	if len(password) == 0 {
+		return createInternalServerError(websites.ErrorCodeWalletPasswordEmpty, err.Error())
```

Comment:
Are you sure there is an error here ?

Reply:
I'm not sure about the question, but the target was to upgrade the error wich was the same as "wrong password" to have a an error specialy when the password is Empty. And then when password is empty it display an "Empty password" error. 

If it's needed to make some adjustments let me know